addons/spreadsheet/static/src/chart/odoo_chart/odoo_line_chart.js

```diff
@@ -122,6 +123,11 @@ function getLineConfiguration(chart, labels, locale) {
             ticks: {
                 color: fontColor,
                 // y axis configuration
+                callback: (value) =>
+                    formatValue(value, {
+                        locale,
+                        format: Math.abs(value) >= 1000 ? "#,##" : undefined,
```

Comment:
Is the `Math.abs(value) >= 1000` really needed ?

Reply:
Not really, it will also work with `format: "#,##"`. However, I followed this approach because the logic in the `o-spreadsheet` library is written this way—likely to save extra calculations (`applyInternalFormat` and `applyInternalNumberFormat`) in cases where the format isn’t required.